suite/package.json

```diff
@@ -10,60 +10,63 @@
   },
   "devDependencies": {
     "@skypack/package-check": "^0.2.2",
-    "@types/aws-lambda": "^8.10.123",
+    "@types/aws-lambda": "^8.10.146",
     "@types/luxon": "^2.4.0",
-    "@types/node": "^16.18.57",
+    "@types/node": "^16.18.122",
     "eslint-config-3box": "^1.0.0",
-    "prettier": "^3.0.3",
-    "semver": "^7.3.5",
-    "ts-loader": "^9.4.4"
+    "prettier": "^3.4.2",
+    "semver": "^7.6.3",
+    "ts-loader": "^9.5.1"
   },
   "dependencies": {
-    "@aws-sdk/client-api-gateway": "^3.427.0",
-    "@aws-sdk/client-dynamodb": "^3.427.0",
-    "@aws-sdk/client-lambda": "^3.427.0",
-    "@ceramicnetwork/codecs": "^5.2.0",
-    "@ceramicnetwork/common": "^6.2.0",
-    "@ceramicnetwork/common-test-utils": "^4.2.0",
-    "@ceramicnetwork/http-client": "^6.2.0",
-    "@ceramicnetwork/stream-model": "^5.2.0",
-    "@ceramicnetwork/stream-model-instance": "^5.2.0",
-    "@ceramicnetwork/stream-tile": "^6.2.0",
+    "@aws-sdk/client-api-gateway": "^3.713.0",
+    "@aws-sdk/client-dynamodb": "^3.713.0",
+    "@aws-sdk/client-lambda": "^3.713.0",
+    "@ceramicnetwork/codecs": "^5.4.0",
+    "@ceramicnetwork/common": "^6.4.0",
+    "@ceramicnetwork/common-test-utils": "^4.4.0",
+    "@ceramicnetwork/http-client": "^6.4.0",
+    "@ceramicnetwork/stream-model": "^5.4.0",
+    "@ceramicnetwork/stream-model-instance": "^5.4.0",
+    "@ceramicnetwork/stream-tile": "^6.4.0",
     "@ceramicnetwork/streamid": "^5.6.0",
     "@composedb/client": "^0.8.0",
     "@composedb/devtools": "^0.8.0",
     "@didtools/pkh-ethereum": "^0.5.0",
-    "@ipld/dag-cbor": "^7.0.0",
-    "@ipld/dag-json": "^10.1.7",
+    "@ipld/dag-cbor": "^7.0.3",
+    "@ipld/dag-json": "^10.2.3",
     "@jest/globals": "^29.7.0",
     "@jest/reporters": "^29.7.0",
     "@stablelib/random": "^1.0.2",
     "axios": "^0.24.0",
     "caip": "^1.1.1",
     "cartonne": "^3.0.1",
-    "codeco": "^1.2.1",
+    "codeco": "^1.4.3",
     "cross-eventsource": "^1.0.0",
     "cross-fetch": "^4.0.0",
     "did-session": "^3.1.0",
-    "dids": "5.0.2",
-    "dotenv": "^16.3.1",
-    "ethers": "^6.13.0",
+    "dids": "5.0.3",
+    "dotenv": "^16.4.7",
+    "ethers": "^6.13.4",
     "graphql-request": "^6.1.0",
     "jest": "^29.7.0",
     "key-did-provider-ed25519": "4.0.2",
     "key-did-resolver": "4.0.0",
     "luxon": "3.2.1",
-    "multiformats": "^13.0.1",
+    "multiformats": "^13.3.1",
     "multihashes-sync": "^2.0.0",
     "node-jq": "^2.3.5",
     "rxjs": "^7.8.1",
-    "ts-jest": "^29.1.1",
+    "ts-jest": "^29.2.5",
     "typescript": "^4.9.5",
-    "uint8arrays": "^5.0.1"
+    "uint8arrays": "^5.1.0"
   },
   "pnpm": {
     "resolutions": {
       "graphql": "16.9.0"
+    },
+    "overrides": {
+        "@graphql-tools/utils": "10.6.3"
```

Comment:
I'm guessing this is the main thing that fixes the tests?

Reply:
Yes, they released a broken version. Had to pin to the previous version.